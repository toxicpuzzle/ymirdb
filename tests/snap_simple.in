SET a 1
SNAPSHOT
SET b 2
SNAPSHOT
CHECKOUT 1
LIST ENTRIES
CHECKOUT 2
LIST ENTRIES
bye