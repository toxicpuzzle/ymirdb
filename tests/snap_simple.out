> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> a [1]

> ok

> b [2]
a [1]

> bye
